release-notes/index.dita

```diff
@@ -86,61 +86,139 @@
             <xref href="https://github.com/dita-ot/dita-ot/issues/3806" format="html" scope="external">#3806</xref>
-          </li>
-          <li id="3807">Remove PDF2 AXF and RenderX in favour of separate repo
+            <fig id="fig_jq5_h5h_qrb">
+              <title>Pre-3.7</title>
+              <codeblock id="codeblock_tcz_g5h_qrb">&lt;xsl:template name="commonattributes">
+  &lt;!-- whole copy of commonattributes named template with customizations -->
+&lt;/xsl:template></codeblock>
+            </fig><fig id="fig_uwr_j5h_qrb">
+              <title>3.7</title>
+              <codeblock id="codeblock_vwr_j5h_qrb">&lt;xsl:template match="@* | node()" mode="commonattributes">
+  &lt;xsl:param name="default-output-class" as="xs:string*"/>
+  &lt;xsl:next-match>
+    &lt;xsl:with-param name="default-output-class" select="$default-output-class"/>
+  &lt;/xsl:next-match>
+  &lt;!-- customizations -->
+&lt;/xsl:template></codeblock>
+            </fig></li>
+          <li id="3807">The source code for Antenna House Formatter (AXF) and RenderX XEP has been removed from the core
+            repository and extracted to dedicated code repositories. The plug-ins remain available and may be installed.
             <xref href="https://github.com/dita-ot/dita-ot/issues/3807" format="html" scope="external">#3807</xref>
           </li>
+          <li>HTML5
+            <ul id="ul_opc_3qx_prb">
+              <li id="3770">The order of linked CSS stylesheets in <xmlelement>head</xmlelement> was changed to load
+                custom CSS last to facilitate overrides.
+                <xref href="https://github.com/dita-ot/dita-ot/issues/3770" format="html" scope="external">#3770</xref>
+              </li>
+              <li id="3738">The <xmlelement>head</xmlelement> element of the HTML template files was simplified by
+                removing <codeph>&lt;meta http-equiv="content-type" content="text/html; charset=UTF-8"></codeph>. This
+                is an older method to specify a character set. A second, simpler <xmlelement>meta</xmlelement> element
+                was also present that defined the UTF-8 character set. The simpler definition remains. The test source
+                files were similarly updated.
+                <xref href="https://github.com/dita-ot/dita-ot/issues/3738" format="html" scope="external">#3738</xref>
+              </li>
+              <li id="3791">Support for Adobe Flash was removed.
+                <xref href="https://github.com/dita-ot/dita-ot/issues/3791" format="html" scope="external">#3791</xref>
+              </li>
+              <li id="3794">The HTML5 stylesheets were updated to use XSL modes instead of named templates. This is a
+                backwards compatible change, however, existing plug-ins should be changed to use modes instead of named
+                templates for:
+                <xref href="https://github.com/dita-ot/dita-ot/issues/3794" format="html" scope="external">#3794</xref>
```

Comment:
We should move this to the end of the first sentence so it doesn't come between the stem sentence colon and the following list.